docs/concepts/configuration/assign-pod-node.md

```diff
@@ -298,231 +298,7 @@ Highly Available database statefulset has one master and three replicas, one may
-support for representing node problems (currently only "node unreachable" and
-"node not ready", corresponding to the NodeCondition "Ready" being "Unknown" or
-"False" respectively) as taints. When the `TaintBasedEvictions` alpha feature
-is enabled (you can do this by including `TaintBasedEvictions=true` in `--feature-gates`, such as
-`--feature-gates=FooBar=true,TaintBasedEvictions=true`), the taints are automatically
-added by the NodeController and the normal logic for evicting pods from nodes
-based on the Ready NodeCondition is disabled.
-(Note: To maintain the existing [rate limiting](/docs/concepts/architecture/nodes/)
-behavior of pod evictions due to node problems, the system actually adds the taints
-in a rate-limited way. This prevents massive pod evictions in scenarios such
-as the master becoming partitioned from the nodes.)
-This alpha feature, in combination with `tolerationSeconds`, allows a pod
-to specify how long it should stay bound to a node that has one or both of these problems.
-
-For example, an application with a lot of local state might want to stay
-bound to node for a long time in the event of network partition, in the hope
-that the partition will recover and thus the pod eviction can be avoided.
-The toleration the pod would use in that case would look like
-
-```yaml
-tolerations:
-- key: "node.alpha.kubernetes.io/unreachable"
-  operator: "Exists"
-  effect: "NoExecute"
-  tolerationSeconds: 6000
-```
-
-(For the node not ready case, change the key to `node.alpha.kubernetes.io/notReady`.)
-
-Note that Kubernetes automatically adds a toleration for
-`node.alpha.kubernetes.io/notReady` with `tolerationSeconds=300`
-unless the pod configuration provided
-by the user already has a toleration for `node.alpha.kubernetes.io/notReady`.
-Likewise it adds a toleration for
-`node.alpha.kubernetes.io/unreachable` with `tolerationSeconds=300`
-unless the pod configuration provided
-by the user already has a toleration for `node.alpha.kubernetes.io/unreachable`.
-
-These automatically-added tolerations ensure that
-the default pod behavior of remaining bound for 5 minutes after one of these
-problems is detected is maintained.
-The two default tolerations are added by the [DefaultTolerationSeconds
-admission controller](https://git.k8s.io/kubernetes/plugin/pkg/admission/defaulttolerationseconds).
-
-[DaemonSet](/docs/concepts/workloads/controllers/daemonset/) pods are created with
-`NoExecute` tolerations for `node.alpha.kubernetes.io/unreachable` and `node.alpha.kubernetes.io/notReady`
-with no `tolerationSeconds`. This ensures that DaemonSet pods are never evicted due
-to these problems, which matches the behavior when this feature is disabled.
+Node affinity is a property of *pods* that *attracts* them to a set of nodes
+(either as a preference or a hard requirement).
+You may want to check [Taints](/docs/concepts/configuration/taint-and-toleration/)
+as well, which allow a *node* to *repel* a set of pods.
```

Comment:
To reviewers: Line 301-304 are new. Mainly providing an anchor to the taint-and-toleration article.